### DISCUSSION TITLE
Theme idea

### DISCUSSION THREAD

**Discussion by anon-y** [LivingWithHippos/unchained-android#192] [Ideas]
Great job on the app! If possible, can we get an all black or dark grey theme? That would be awesome.

**Comment by LivingWithHippos** [comment]
Hello, there's already a black and white theme, if it's not what you meant could you give me an example?

**Comment by anon-y** [comment]
> Hello, there's already a black and white theme, if it's not what you meant could you give me an example? 

Oh yeah that's the one I use but I wish if the white parts were dark grey lol 😅 Something less jarring.
